qa/L0_nvtx/test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+ # Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ # 
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+TEST_LOG="./nvtx.log"
+
+
+apt-get update && \
+    apt-get install -y --no-install-recommends python
+
+rm -f $TEST_LOG
+RET=0
+
+set +e
+
+(cd /opt/pytorch/pyprof/qa/L0_nvtx/ && \
```

Comment:
1. The CI is already in this directory. That is how it is running the test.sh file in the first place, so remove this `cd`. 
2. Remove the set +e above so we can see that this script is running. Right now it doesn't look like anything is reunning